lib/index.ts

```diff
@@ -499,6 +507,19 @@ export function emit(rootDecl: TopLevelDeclaration, rootFlags = ContextFlags.Non
                     printObjectTypeMembers(e.members);
                     break;
 
+                case "function":
+                    writeFunctionType(e);

```

Comment:
My intend was to create anonymous function types. I will rework with into a FunctionExpression type at the top-level, okay?